.gitignore

```diff
@@ -7,3 +7,5 @@
 *.o
 *.pdf
 o[1-9]*
+tags
```

Comment:
I put always-ignore patterns like "tags" and "TAGS" into my personal global git ignore file, so I don't have to add it to per-repo ignore files. I don't know how common this practice is.